app/schemes/puzzle_registry.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import Column, Integer, String, Text, TIMESTAMP, UniqueConstraint
+from app.core.database import Base
+from datetime import datetime
+
+class PuzzleRegistry(Base):
+  __tablename__ = "puzzle_registry"
```

Comment:
This could sudoku_entry or sudoku_registry, since different puzzles would have different table, even though we do not care for that scenario for now